CHANGELOG.md

```diff
@@ -60,6 +60,7 @@
         - Checking of cached front page details against database. #2696
         - Inconsistent display of mark private checkbox for staff users
         - Clear user categories when staff access is removed. #2815
+        - Only trigger one change event on initial popstate.
```

Comment:
Maybe this should mention it’s for the `filter_categories` dropdown?

Reply:
And the status one too :) I think anyone who cares can look at the code/git log, want to keep this high level.